subsystems/visionsubsystem.py

```diff
@@ -121,6 +154,15 @@ def updateAdvantagescopePose(
         )
         SmartDashboard.putNumberArray(cameraKey, cameraPose)
 
+    def getNoteToCamera(self, note: PhotonTrackedTarget) -> Pose2d:
+        x = constants.kNoteCameraHeight / tan(
+            constants.kNoteCameraPitch - note.getPitch() * constants.kRadiansPerDegree
+        )
+        dist = (constants.kNoteCameraHeight**2 + x**2) ** 0.5
+        y = dist * tan(-note.getYaw() * constants.kRadiansPerDegree)
+
+        return Pose2d(x, y, atan(y / x))
```

Comment:
since the function says note --> camera it should be a transform2d, not a pose2d